llvm/lib/Object/DXContainer.cpp

```diff
@@ -288,6 +290,51 @@ Error DirectX::RootSignature::parse(StringRef Data) {
                             llvm::Twine(FValue));
   Flags = FValue;
 
+  Current = Begin + RootParametersOffset;
+  for (uint32_t It = 0; It < NumParameters; It++) {
```

Comment:
Better to call this `I` - Usually `It` is used for an iterator of some kind, but this is just an index.